unit-tests/run-unit-tests.py

```diff
@@ -555,6 +560,8 @@ def test_wrapper( test, configuration=None, repetition=1, sns=None ):
                 test_ok = True
                 for repetition in range(repeat):
                     test_ok = test_wrapper( test, repetition = repetition ) and test_ok
+                    if repetition + 1 < repeat:
```

Comment:
I'd rather move this one line up and say:
`if repetition > 0:`
Same on line 588

Reply:
Why not put this inside test_wrapper?

Reply:
You mean the complete for loop?

Reply:
No, the debug log